### PR TITLE
Splitting data products into a small and large version.

### PR DESCRIPTION
**Pull request by cmccully**

### CODE REVIEW

File: banzai_nres/tests/test_extract.py [hunk 1]
```diff
@@ -157,14 +157,13 @@ def five_hundred_square_image(maxflux, number_traces, trace_width, read_noise=10
...
-                                                         blaze={'id': np.arange(number_traces) + 1,
-                                                                'blaze': [np.ones(traces.shape[1]) for i in
-                                                                          range(number_traces)],
-                                                                'blaze_error': [np.ones(traces.shape[1]) for i in
-                                                                                range(number_traces)]},
-                                                         fibers={'fiber': np.arange(number_traces) + 1,
-                                                                 'order': np.arange(number_traces) + 1})], 'foo.fits')
+    image = NRESObservationFrame([CCDData(data=data, uncertainty=uncertainty, meta={'OBJECTS': 'tung&tung&none'})],
+                                 'foo.fits')
+    image.traces = traces,
```
Comment (captain-exoplanet) [inline banzai_nres/tests/test_extract.py:162]: The comma at the end of this line seems out of place

Comment (captain-exoplanet) [reply banzai_nres/tests/test_extract.py:162]: Though it's possible I just don't understand the new syntax

File: banzai_nres/frames.py [hunk 2]
```diff
@@ -99,95 +116,106 @@ def science_fiber(self):
...
-    @property
-    def features(self):
-        return self.primary_hdu.features
-
-    @features.setter
-    def features(self, value):
-        self.primary_hdu.features = value
-
-    @property
-    def line_list(self):
```
Comment (captain-exoplanet) [inline banzai_nres/frames.py:161]: How do we handle the line list now? It looks like all the material defining how the line list is handled has been deleted but the parts of wavelength.py that actually fill and use the line list are unchanged. Will this still work with all of the changes in frames.py?

File: banzai_nres/tests/test_extract.py (hunk 1 above)
Comment (mgdaily) [reply banzai_nres/tests/test_extract.py:162]: I agree. Extra comma here seems bad

Comment (cmccully) [reply banzai_nres/tests/test_extract.py]: Agreed. That was a typo that was causing weird issues that I have now fixed.

File: banzai_nres/frames.py (hunk 2 above)
Comment (cmccully) [reply banzai_nres/frames.py:161]: Yes. The line list will still work correctly. The only difference is that it is not attached to the output data products anymore.

**Review by captain-exoplanet — APPROVED** [review]
Everything looks good. Approved!

**Pull request merged**